apps/prairielearn/src/lib/externalGraderSqs.js

```diff
@@ -55,14 +48,10 @@ export class ExternalGraderSqs {
           };
 
           const s3 = new S3(makeS3ClientConfig());
-          await new Upload({
-            client: s3,
-            params,
-          }).done();
-        },
-        async () => {
+          await new Upload({ client: s3, params }).done();
+
           // Store S3 info for this job
-          sqldb.queryAsync(sql.update_s3_info, {
+          await sqldb.queryAsync(sql.update_s3_info, {
```

Comment:
@nwalters512 (or @mwest1066 in his absence) this query code was not awaiting, I added an await. Can you double check that this was indeed an error (instead of code that ran in the background on purpose)? I don't have an easy way to test S3 handling of external graders.

Reply:
I agree that this was an error without the `await`.